core/src/Extensions.cs

```diff
@@ -28,8 +28,9 @@
 using System;
 using System.Globalization;
 using System.Text;
+using Microsoft.Identity.Client;
 
-namespace Microsoft.Identity.Client.Internal
+namespace Microsoft.Identity.Core
 {
     internal static class Extensions
```

Comment:
>Extensions [](start = 26, length = 10)

Here we have another set of extensions. should this be placed in a sub folder called extensions together with the helper class which is also basically a set of extensions?

Reply:
I have moved this class in helpers namespace and renamed to ClassExtensions. I have also moved all extension methods into this class

---
In reply to: [161031942](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161031942) [](ancestors = 161031942)